exit_db/migrations/2022-05-16-200950_ipv6_migration/up.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE clients
+    ADD COLUMN internet_ipv6 varchar(132) NOT NULL DEFAULT ''
+;
+CREATE TABLE assigned_ips
+(
+    subnet varchar(132) CONSTRAINT secondkey PRIMARY KEY,
+    available_subnets varchar(512) NOT NULL,
+    iterative_index bigint DEFAULT 0 NOT NULL
```

Comment:
I'm probably just not up to speed on postgres but how can we have only one interative index to keep track of assigned ip's across many subnets.

Similar what is in the subnet field if we may have many different available subnets?

Reply:
👍🏻  ok so this has multiple rows that's what I was missing. Thanks.